CHANGELOG.md

```diff
@@ -567,334 +491,490 @@ Single File I/O:
-  - `random_seq_mod:random_inverse_gamma() 'shape' to 'rshape', 'scale' to 'rscale'`
-  - `obs_def_mod:init_obs_def() 'kind' to 'obkind', 'time' to 'obtime'`
-  - `obs_utilities_mod: 'start' to 'varstart', 'count' to 'varcount'`
+    This shows where inflation file names are defined.
+      > grep -I set_file_metadata */*.f90 | grep inf
+    filter/filter_mod.f90:
+       call set_file_metadata(file_info, PRIOR_INF_MEAN, stage, 'priorinf_mean', 'prior inflation mean')
+       call set_file_metadata(file_info, PRIOR_INF_SD,   stage, 'priorinf_sd',   'prior inflation sd')
+       call set_file_metadata(file_info, POST_INF_MEAN,  stage, 'postinf_mean',  'posterior inflation mean')
+       call set_file_metadata(file_info, POST_INF_SD,    stage, 'postinf_sd',    'posterior inflation sd')
 
-- The **FESOM** model is now Manhattan-ready. Thanks to **Ali Aydogdu**
+    subroutine set_member_file_metadata(file_info, ens_size, my_copy_start)
+       call set_file_metadata(file_info, icopy, stage_name, base_name, desc, offset)
 
-- The **noah** model is now Manhattan-ready and may be used with NOAH-MP.
+    subroutine set_stage_file_metadata(file_info, copy_number, stage, base_name, desc, offset)
+       write(string1,'(A,''.nc'')') trim(stage_name)//'_'//trim(base_name)
 
-- bugfixed references to the `documentation` directory that was renamed
-  `docs` to comply with GitHub Pages.
+    subroutine set_explicit_file_metadata(file_info, cnum, fnames, desc)
+       file_info%stage_metadata%filenames(cnum,idom)        = trim(fnames(idom))
+       file_info%stage_metadata%file_description(cnum,idom) = trim(string1)
 
-- improved `test_dart.csh` functionality.
+    function construct_file_names(file_info, ens_size, copy, domain)
+       write(construct_file_names, '(A, ''_member_'', I4.4, A, ''.nc'')') &
+                           trim(file_info%root_name), copy, trim(dom_str)
 
-------------------------------------------------------------------------------
-## MMM DD YYYY :: summary of changes in next Manhattan update       Tag: vX.Y.Z
+Also see
+   harnesses/filename_harness/files:  ENS_MEAN_COPY       PriorDiag_mean.nc
+```
 
-- Added the MPD observation converter for ABSOLUTE_HUMIDITY observations.
-  Thanks to Michael Ying for contributing it.
+#### ADDITIONAL NOTES :
 
-- Converted CHANGELOG to a markdown document.
+1. currently the closest_member_tool is broken but plans on being fixed soon.
+1. restart_file_tool and most model_to_dart/dart_to_model programs have been
+  deprecated, since DART formated restarts are no longer supported.
+1. some programs such as model_mod_check have not been fully tested and need
+  to be exercised with the new naming conventions.
 
-- Converted `observations/obs_converters/observations.html` to `observations/obs_converters/README.md`
+------------------------------------------------------------------------------
+## MMM DD YYYY :: summary of changes in next update       Tag: vX.Y.Z
```

Comment:
you might want to remove this line.  updates aren't at the end anymore and it's pretty clear what the formats are now.

instead, maybe add a line here saying something like:  To see previous update history use: git xxx  (repeat the command you gave at the top about using git log).